lib/capybara/node/finders.rb

```diff
@@ -166,7 +166,13 @@ def all(*args)
       # @return [Capybara::Node::Element]            The found element or nil
       #
       def first(*args)
+        if Capybara.first_default_waiting
+          options = if args.last.is_a?(Hash) then args.pop.dup else {} end
+          args.push({minimum: 1}.merge(options))
+        end
         all(*args).first
+      rescue Capybara::ExpectationNotMet
+        nil
```

Comment:
If passing options It does, but not for the common case of node.first().  the documentation states it returns the element or nil if none exists, so I would say the previous behavior with options is actually a bug. Also without this. Node.first() would raise if Capybara.wait_on_first... Was true, which again doesn't match the docs, and would break many more tests (since most people don't realize their are options that can be passed). What use case do you have where you would want #first to raise?